zeroeq/receiver.cpp

```diff
@@ -180,6 +177,22 @@ Receiver::~Receiver()
     _impl->remove(this);
 }
 
+Receiver::Receiver(Receiver&& from)
+    : _impl(from._impl)
+{
+    from._impl.reset();
+}
+
+Receiver& Receiver::operator=(Receiver&& from)
```

Comment:
It can't, because otherwise the Impl needs to be defined in the declaration since this._impl might need to be destructed.